spec/factories/linkage_group.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :linkage_group do
-    sequence(:linkage_group_label) { Faker::Lorem.characters(10) }
+    sequence(:linkage_group_label) { Faker::Lorem.characters(20) }
```

Comment:
Instead of extending the string please use the sequence "n" value, as it is done in design_factor factory. Then, it is enough to have it shorter.